elements/rh-button/rh-button.css

```diff
@@ -0,0 +1,480 @@
+
+  /* SECONDARY */
+  --_secondary-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_secondary-background-color: transparent;
+  --_secondary-border-color: #2578bb; /* WARNING: not a token value */
+  --_secondary-border-width: var(--rh-border-width-sm, 1px);
+  --_secondary-active-color: var(--rh-color-interactive-blue-lighter, #73bcf7);
+  --_secondary-active-background-color: transparent;
+  --_secondary-active-border-color: var(--rh-color-interactive-blue-lighter, #73bcf7);
+  --_secondary-active-border-width: var(--rh-border-width-md, 2px);
+  --_secondary-focus-color: var(--rh-color-interactive-blue-lighter, #73bcf7);
+  --_secondary-focus-background-color: transparent;
+  --_secondary-focus-border-color: var(--rh-color-interactive-blue-lighter, #73bcf7);
+  --_secondary-focus-border-width: var(--rh-border-width-md, 2px);
+  --_secondary-hover-color: var(--rh-color-interactive-blue-lighter, #73bcf7);
+  --_secondary-hover-background-color: transparent;
+  --_secondary-hover-border-color: var(--rh-color-interactive-blue-lighter, #73bcf7);
+  --_secondary-hover-border-width: var(--rh-border-width-md, 2px);
+
+  /* TERTIARY */
+  --_tertiary-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_tertiary-background-color: transparent;
+  --_tertiary-border-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_tertiary-border-width: var(--rh-border-width-sm, 1px);
+  --_tertiary-active-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_tertiary-active-background-color: transparent;
+  --_tertiary-active-border-color: var(--rh-color-border-strong-on-dark, #ffffff);
+  --_tertiary-active-border-width: var(--rh-border-width-md, 2px);
+  --_tertiary-focus-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_tertiary-focus-background-color: transparent;
+  --_tertiary-focus-border-color: var(--rh-color-border-strong-on-dark, #ffffff);
+  --_tertiary-focus-border-width: var(--rh-border-width-md, 2px);
+  --_tertiary-hover-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_tertiary-hover-background-color: transparent;
+  --_tertiary-hover-border-color: var(--rh-color-border-strong-on-dark, #ffffff);
+  --_tertiary-hover-border-width: var(--rh-border-width-md, 2px);
+
+  /* LINK */
+  --_link-color: var(--rh-color-interactive-blue-lighter, #73bcf7);
+  --_link-background-color: transparent;
+  --_link-active-color: var(--rh-color-interactive-blue-lightest, #bee1f4);
+  --_link-active-background-color: transparent;
+  --_link-focus-color: var(--rh-color-interactive-blue-lightest, #bee1f4);
+  --_link-focus-background-color: transparent;
+  --_link-focus-outline-color: var(--rh-color-interactive-blue-lighter, #73bcf7);
+  --_link-hover-color: var(--rh-color-interactive-blue-lightest, #bee1f4);
+  --_link-hover-background-color: transparent;
+
+  /* CLOSE */
+  --_close-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_close-background-color: transparent;
```

Comment:
Just noticed that there's a blue background that appears on hover for the `close` button in dark theme. I don't _think_ that's supposed to be there. Maybe the icon color can change to `#D2D2D2`. This should have enough contrast against our three dark surface colors.

Reply:
That was a request from @nikkimk

Reply:
So @marionnegp and I chatted and what we think the outline on _focus_ should be sufficient on its own, but there is some adjustment that needs to be made for _hover_ contrast.

Reply:
Yep, and it was for the `link` variant, instead of the `close` button.

Let's go ahead and remove the bg color on focus for `close` in dark theme.